packages/nodes-from-anchor/src/v01/AccountNode.ts

```diff
@@ -0,0 +1,44 @@
+import {
+    AccountNode,
+    accountNode,
+    assertIsNode,
+    bytesTypeNode,
+    camelCase,
+    fieldDiscriminatorNode,
+    structFieldTypeNode,
+    StructTypeNode,
+    structTypeNode,
+} from '@kinobi-so/nodes';
+
+import { getAnchorDiscriminatorV01 } from '../discriminators';
+import { IdlV01Account, IdlV01TypeDef, IdlV01TypeDefTyStruct } from './idl';
+import { structTypeNodeFromAnchorV01 } from './typeNodes';
+
+export function accountNodeFromAnchorV01(idl: IdlV01Account, types: IdlV01TypeDef[]): AccountNode<StructTypeNode> {
+    const definition = types.find(type => type.name === idl.name) as IdlV01TypeDef;
+
+    const idlName = definition.name;
+    const name = camelCase(idlName);
+
+    // TODO: support enum and generic types.
+    let data = structTypeNodeFromAnchorV01(definition.type as IdlV01TypeDefTyStruct);
+    assertIsNode(data, 'structTypeNode');
+
+    // Account discriminator.
+    const discriminator = structFieldTypeNode({
+        defaultValue: getAnchorDiscriminatorV01(idl.discriminator),
+        defaultValueStrategy: 'omitted',
+        name: 'discriminator',
+        type: bytesTypeNode(),
+    });
+
+    data = structTypeNode([discriminator, ...data.fields]);
+    const discriminators = [fieldDiscriminatorNode('discriminator')];
```

Comment:
I see this has changed since yesterday but this was perfect. That is: copying over the `StructTypeNode` of the linked `DefinedTypeNode` and prepending the discriminator. 👌